include/nccl_ofi_rdma.h

```diff
@@ -361,6 +361,7 @@ typedef struct nccl_net_ofi_rdma_req {
  */
 typedef struct nccl_ofi_rdma_ep_name {
 	char ep_name[MAX_EP_ADDR];
+	size_t ep_name_len;
```

Comment:
I like storing `ep_name_len` in the same struct as `ep_name`. But this will cause some extra padding since `MAX_EP_ADDR` is not a multiple of 8.

This is not fast-path code so I didn't feel strongly about it.

Reply:
i wouldn't worry about it.